guide/how-it-works/introduction.md

```diff
@@ -8,11 +8,16 @@ permalink: /guide/how-it-works/
 main_classes: -no-top-padding
 ---
 
-<!--
-
-Illustration sources:
-
--->
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/how-it-works.png"
+   retina = "/assets/images/guide/how-it-works/how-it-works-@2x.png"
+   mobile = "/assets/images/guide/how-it-works/how-it-works-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/how-it-works-mobile@2x.png"
+   alt-text = "How it works header image."
```

Comment:
```suggestion
   alt-text = "Multiple interlocking gears, center gear has a bitcoin symbol on it"
```
Alt text should describe what is in the image for people (and computers) who can't see it.